erpnext/setup/install.py

```diff
@@ -29,12 +29,12 @@ def after_install():
 
 
 def check_setup_wizard_not_completed():
-	if frappe.db.get_default('desktop:home_page') == 'desktop':
-		print()
-		print("ERPNext can only be installed on a fresh site where the setup wizard is not completed")
-		print("You can reinstall this site (after saving your data) using: bench --site [sitename] reinstall")
-		print()
-		return False
+	if frappe.db.get_default('desktop:home_page') != 'setup-wizard':
+		message = """Cannot install ERPNext.
+
+ERPNext can only be installed on a fresh site where the setup wizard is not completed.
+You can reinstall this site (after saving your data) using: bench --site [sitename] reinstall"""
```

Comment:
The line "Cannot install ERPNext." should be handled by Frappe IMO.

```suggestion
		message = """ERPNext can only be installed on a fresh site where the setup wizard is not completed. 
You can reinstall this site (after saving your data) using: bench --site [sitename] reinstall"""